static_report/public/index.html

```diff
@@ -1,89 +1,76 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta name="viewport" content="initial-scale=1.0, width=device-width" />
-    <link
-      rel="apple-touch-icon"
+    <meta name="viewport" content="initial-scale=1.0, width=device-width" >
+    <link       rel="apple-touch-icon"
       sizes="57x57"
       href="logo/apple-icon-57x57.png"
-    />
-    <link
-      rel="apple-touch-icon"
+    >
+    <link       rel="apple-touch-icon"
       sizes="60x60"
       href="logo/apple-icon-60x60.png"
-    />
-    <link
-      rel="apple-touch-icon"
+    >
+    <link       rel="apple-touch-icon"
       sizes="72x72"
       href="logo/apple-icon-72x72.png"
-    />
-    <link
-      rel="apple-touch-icon"
+    >
+    <link       rel="apple-touch-icon"
       sizes="76x76"
       href="logo/apple-icon-76x76.png"
-    />
-    <link
-      rel="apple-touch-icon"
+    >
+    <link       rel="apple-touch-icon"
       sizes="114x114"
       href="logo/apple-icon-114x114.png"
-    />
-    <link
-      rel="apple-touch-icon"
+    >
+    <link       rel="apple-touch-icon"
       sizes="120x120"
       href="logo/apple-icon-120x120.png"
-    />
-    <link
-      rel="apple-touch-icon"
+    >
+    <link       rel="apple-touch-icon"
       sizes="144x144"
       href="logo/apple-icon-144x144.png"
-    />
-    <link
-      rel="apple-touch-icon"
+    >
+    <link       rel="apple-touch-icon"
       sizes="152x152"
       href="logo/apple-icon-152x152.png"
-    />
-    <link
-      rel="apple-touch-icon"
+    >
+    <link       rel="apple-touch-icon"
       sizes="180x180"
       href="logo/apple-icon-180x180.png"
-    />
-    <link
-      rel="icon"
+    >
+    <link       rel="icon"
       type="image/png"
       sizes="192x192"
       href="logo/android-icon-192x192.png"
-    />
-    <link
-      rel="icon"
+    >
+    <link       rel="icon"
       type="image/png"
       sizes="32x32"
       href="logo/favicon-32x32.png"
-    />
-    <link
-      rel="icon"
+    >
+    <link       rel="icon"
       type="image/png"
       sizes="96x96"
       href="logo/favicon-96x96.png"
-    />
-    <link
-      rel="icon"
+    >
+    <link       rel="icon"
       type="image/png"
       sizes="16x16"
       href="logo/favicon-16x16.png"
-    />
-    <link rel="manifest" href="logo/manifest.json" />
-    <meta name="msapplication-TileColor" content="#ffffff" />
-    <meta name="msapplication-TileImage" content="/ms-icon-144x144.png" />
-    <meta name="theme-color" content="#ffffff" />
+    >
+    <link rel="manifest" href="logo/manifest.json" >
+    <meta name="msapplication-TileColor" content="#ffffff" >
+    <meta name="msapplication-TileImage" content="/ms-icon-144x144.png" >
+    <meta name="theme-color" content="#ffffff" >
```

Comment:
Just curious, these changes are changing by yourself (handy)? Or there are changes by scripts?

Reply:
It's changed by script when i run `yarn build`. Is it because of different prettier settings?

Reply:
The html is generated by this file

https://github.com/InfuseAI/piperider/blob/main/static_report/src/sdlc/core.js

Reply:
Got it, thanks!